TWUITests/Helpers/XCUIElement+Extensions.swift

```diff
@@ -49,14 +49,55 @@ public extension XCUIElement {
      Parameters:
      - element: XCUIElement in scroll view to be found
      - maxSwipes: maximum swipe up actions
-     Return: true if element was found while swiping up
+     Return:
+       true if element was found while swiping up
      */
     func swipeUp(to element: XCUIElement, maxSwipes: UInt = 5) -> Bool {
-        for _ in 0..<maxSwipes {
+        find(element: element, maxActions: maxSwipes, action: self.swipeUp() )
+    }
+
+    /**
+     Scroll Left in scroll view until XCUIElement is found or maximum swipe
```

Comment:
Looks good, just comment is a bit unclear, I think. Especially `or maximum swipe`.

Reply:
Will update 👍